extras/systemd/dynflow-sidekiq@.service

```diff
@@ -15,7 +15,7 @@ Environment=REDIS_PROVIDER=DYNFLOW_REDIS_URL
 # https://www.mikeperham.com/2018/04/25/taming-rails-memory-bloat/
 Environment=MALLOC_ARENA_MAX=2
 WorkingDirectory=/usr/share/foreman
-ExecStart=/usr/bin/sidekiq -e ${RAILS_ENV} -r ${DYNFLOW_SIDEKIQ_SCRIPT} -C /etc/foreman/dynflow/%i.yml
+ExecStart=/usr/share/foreman/bin/sidekiq -e ${RAILS_ENV} -r ${DYNFLOW_SIDEKIQ_SCRIPT} -C /etc/foreman/dynflow/%i.yml
```

Comment:
Shouldn't the path be `/usr/share/foreman/bin/foreman-sidekiq` instead of `/usr/share/foreman/bin/sidekiq`?

Reply:
I have renamed this in core to comply with what's already in the directory (no foreman prefix). Where do you still see it?

Reply:
Oh, looks like I've got it backwards. Then this https://github.com/theforeman/foreman-packaging/pull/5255/files#diff-3399e5c03dc282964481f8f3ced9c141R654 should probably say `%{_datadir}/%{name}/bin/sidekiq`

Reply:
Good point, I am dropping the first line since @ekohl made a good point the whole bin/ directory is added in the main package already.

Need to correct SELinux file context file, I forgot actually!